modules/apps/frontend-js/frontend-js-react-web/npmscripts.config.js

```diff
@@ -13,6 +13,17 @@
  */
 
 module.exports = {
-	federation: true,
+	federation: {
+		bridges: [
+			'classnames',
+			'formik',
+			'prop-types',
+			'react',
+			'react-dnd',
+			'react-dnd-html5-backend',
+			'react-dom',
+		],
+		runLegacyBuild: false,
+	},
 	preset: '@liferay/npm-scripts/src/presets/standard',
```

Comment:
Is the preset needed?

Reply:
AFAIK, yes. It's how npm-script seems to work...